src/en/proposals/dragonjspider-Wawaspecies.md

```diff
@@ -0,0 +1,64 @@
+# Wawa, or Scugs as a playable roundstart species.
+
+
+| Designers | Implemented | GitHub Links |
+|---|---|---|
+| fairlysadpanda, erisws, cornerdogs, dragonjspider, kiterny, thecatbastard | :information_source: Open PR | TBD |
+
+
+## Overview
+This proposal's plan is to add "Wawa" (Inspired by Scugs) as a playable roundstart species, like lizards or humans. While that is our main goal, if the option is presented instead, a ghost role for them would be nice as well.
+This doc will go into detail about their stats and abilities, and discussing the possible impacts some of these may have in different scenarios.
+Feedback would be greatly appreciated; as there are also some experimental ideas that may be unbalanced, or unfun to play around.
+
+## Background
+We have discussed this a lot on the discord, leading to this proposal (starting from https://discord.com/channels/310555209753690112/1212937078153805844/1220355368228618310). There have been a lot of ideas behind this, and there may be more that were overlooked.
+Feel free to read all of it, but this doc will also summarise what were agreed to be good ideas, as well as the more experimental/system effecting changes. We also understand that there may be some copyright issues with this, so I will attach any permissions to this as well, If and when any are given.
+
+## Copyright permissions
+Currently we have permission from a dev of Downpour, expeditions specifically. Whilst this isn't "proper" approval, more of a semi-approval, we believe it allows us to continue with a project of making a "Scug Inspired" species. The following screenshots are provided courtesy of fairlysadpanda.
+
+![image](https://github.com/Dragonjspider/Wawa-Player-Species-/assets/140023808/ae7437cf-f4d3-42c1-8798-b8ce24a2725f)
+![image](https://github.com/Dragonjspider/Wawa-Player-Species-/assets/140023808/69d00309-faae-41e0-a6aa-9e2916052de3)
+
+
+
+## Species
+Our Official name for the species would be Wawa (proposed by fairlysadpanda). Whilst this isn't exactly "creative" for them, we believe a distinct but still related name suits them. We're not going to be naming them Scug(s), or Slugcat(s), because of more specific issues with copyright and such, and lack of explicit written permission.
+For personalization they will be able to have most colors for skin, similar to lizards or moths.
```

Comment:
I think just having skin colors for customization is kind of boring. Even basic downpour slugcats have some more interesting stuff going on- open the floodgates for the slug OCs you've always wanted to see, in my opinion.

Reply:
Yea I kept it basic here because detailed stuff would be more on whoever does the sprites, I couldn't find the words to say they would have more customization options

Reply:
Like Dragon says, most of the customization will be up to Spriters, and also we can retroactively add additional customization options.